lib/core/presentation/common_widgets/appBar.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+
+class MyAppBar extends AppBar {
+  String apptitle = 'Get The Job!';
```

Comment:
it's not an `apptitle`, it's `appBarTitle` or `pageTitle`

please make it as final: 
`final String apptitle;`

and later in the constructor:

`MyAppBar({this.apptitle = 'Get The Job!'})`